include/Util/helper.php

```diff
@@ -249,3 +249,37 @@ function get_advanced_search_url() {
 
 	return add_query_arg( 'el_as', 1, $admin_url );
 }
+
+/**
+ * Gets the Column labels to be used in LogList table.
+ *
+ * @since 2.3.0
+ *
+ * @param string $db_column
+ *
+ * @return string
+ */
+function get_column_label_by_db_column( $db_column ) {
+	$labels = array(
+		'id'          => __( 'ID', 'email-log' ),
+		'sent_date'   => __( 'Sent at', 'email-log' ),
+		'to'          => __( 'To', 'email-log' ), // EmailLog\Core\UI\ListTable::get_columns() uses `to`
+		'to_email'    => __( 'To', 'email-log' ),
+		'subject'     => __( 'Subject', 'email-log' ),
+		'message'     => __( 'Message', 'email-log' ),
+		'from'        => __( 'From', 'email-log' ),
+		'cc'          => __( 'CC', 'email-log' ),
+		'bcc'         => __( 'BCC', 'email-log' ),
+		'reply-to'    => __( 'Reply To', 'email-log' ),
+		'attachments' => __( 'Attachment', 'email-log' ),
+		'attachment'  => __( 'Attachment', 'email-log' ),
+	);
+
+	// TODO: @sudar, Do we need a filter here to add items to the $labels array?
```

Comment:
@mariadanieldeepak 

Let's add a filter here. Once the filter is added, this PR is good to go.